### DISCUSSION TITLE
There is no way to opt-out the flighting?

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/winget-cli#866] [Q&A] [answered]
# Description of the new feature/enhancement

# Proposed technical implementation details (optional)

**Comment by denelon** [comment, marked as answer]
@odysseus443 I'll convert this to a discussion. That way others can find it.

There is an opt-out form for the "Windows Package Manager Insiders" group.
You can use our [form](https://aka.ms/AppInstaller_Unsubscribe) to unsubscribe.

In order to remove the Windows Package Manager from your system, you will need to install an earlier build of the "App Installer". We're still working to provide a download for the current stable build installed on Windows.

The Windows Package Manager is flighted to all Windows Insiders Builds.

Edited: Adding unsubscribe URL.

**Reply by ghost** [reply]
Where is the link to that group or form?

**Reply by denelon** [reply]
I'm looking it up. I didn't have it bookmarked. I'll also update the documentation as soon as I locate it. I'm sorry for the delay here.

**Reply by ghost** [reply]
Okay, thanks.

**Reply by denelon** [reply]
@odysseus443, I just added the link to unsubscribe. That is the first step in the process.

**Reply by VincentJoshuaET** [reply]
@denelon I have submitted my Microsoft account multiple times and I still seem to be getting preview versions. I have Windows 11 stable release, not an Insider

**Comment by denelon** [comment]
Related discussion at https://github.com/microsoft/winget-cli/discussions/844
